server/app/pyproject.toml

```
@@ -0,0 +1,64 @@
+[build-system]
+requires = [
+    "setuptools>=45",
+    "wheel",
+    "setuptools_scm[toml]>=6.2"
+]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools_scm]
+# Configure setuptools_scm for version management:
+# - Automatically infers the version number from the most recent git tag
+# - Generates a version.py file in the package directory
+# - Allows for automatic versioning between releases (e.g., 1.0.1.dev4+g12345)
+# If you want to use the version anywhere in the code, use
+# ```
+# from app.version import version
+# print(f"Project version: {version}")
+# ```
+root = "../.."  # Defines the path to the root of the repository
+version_file = "version.py"
+
+[project]
+name = "basyx-python-server"
+dynamic = ["version"]
+description = "The Eclipse BaSyx Python Server, an implementation of the BaSyx AAS Server" #FIXME
+authors = [
+    { name = "The Eclipse BaSyx Authors", email = "admins@iat.rwth-aachen.de" }
+]
+readme = "README.md"
+license = { file = "LICENSE" }
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+    "Development Status :: 5 - Production/Stable"
+]
+requires-python = ">=3.9"
+dependencies = [
+    "urllib3>=1.26,<3",
+    "Werkzeug>=3.0.3,<4",
+]
+
+[project.optional-dependencies]
+dev = [
+    "mypy",
+    "pycodestyle",
+    "codeblocks",
+    "schemathesis~=3.7",
+    "jsonschema~=4.7",
+    "hypothesis~=6.13",
+    "lxml-stubs~=0.5.1",
+]
+
+[project.urls]
+"Homepage" = "https://github.com/eclipse-basyx/basyx-python-sdk"
+
+[tool.setuptools]
+packages = { find = { exclude = ["test*"] } }
+
+[tool.setuptools.package-data]
+app = ["py.typed"]
+
+[tool.mypy]
+exclude = "build/"
```

```suggestion
exclude = "build/"

```
Add a newline at the end of the file